assets/scss/_custom.scss

```diff
@@ -61,6 +61,15 @@ body.td-404 main .error-details {
   margin-bottom: 0;
 }
 
+/* Global - Mermaid.js diagrams */
+
+.mermaid {
+  overflow: scroll;
```

Comment:
nit:
`overflow-x: scroll`.
As is, there is an empty vertical scrollbar.

Reply:
```suggestion
  overflow-x: scroll;
```